.github/workflows/ci.yml

```diff
@@ -57,11 +57,41 @@ jobs:
         run: |
           mkdir html
           sphinx-build -b html documentation/library_guide/ html/
+          tar -czvf html.tar.gz html/
```

Comment:
This line creates lots of internal archives in .zip with documentation. It's not needed. Is it possible to remove the redundant archives?